### ISSUE TITLE
Typing a modifier key anywhere removes selected text from an input in 3.5.0

### ISSUE THREAD

**Issue by dwelle** [cypress-io/cypress#5439]
### Current behavior:

Typing a modifier key ~~into an input~~ anywhere removes selected text in an active input.

### Desired behavior:

Shouldn't touch input value at all.

### Steps to reproduce: (app code and test code)

```js
describe('test', () => {
    it('test', () => {
        cy.document().then(doc => {
            doc.body.innerHTML = `
                <textarea class="one">hello</textarea>
                <textarea class="two">hello</textarea>
                <input class="three" value="hello">
            `;
        });

        // (1) clears whole textarea
        cy.get('.one').type('{selectall}').type('{ctrl}');

        // (2) clears whole textarea
        cy.get('.two').type('{selectall}');
        cy.get('body').type('{ctrl}');

        // (3) removes last letter
        cy.get('.three')
            .focus()
            .then( $input => {
                $input[0].selectionStart = 4;
                $input[0].selectionEnd = 5;
            })
            .type('{ctrl}');
    });
});
```
### Versions

- Cypress `3.5.0`, Win 7, chrome stable 77 (headed) & chrome canary 73 (headless)
- It's possible a regression in older cypress version (I've migrated from `3.3.1` → `3.5.0` directly)

**Comment by jennifer-shehane** [comment]
I can confirm that this behavior has changed from 3.4.1 to 3.5.0. 

Test exhibiting change.

```js
describe('test', () => {
  it('test', () => {
    cy.document().then((doc) => {
      doc.body.innerHTML = `
        <textarea class="one">hello</textarea>
        <textarea class="two">hello</textarea>
        <input class="three" value="hello">
      `
    })

    // (1) clears whole textarea
    cy.get('.one').type('{selectall}').type('{ctrl}')

    // (2) clears whole textarea
    cy.get('.two').type('{selectall}')
    cy.get('body').type('{ctrl}')

    // (3) removes last letter
    cy.get('.three')
      .focus()
      .then(($input) => {
        $input[0].selectionStart = 4
        $input[0].selectionEnd = 5
      })
      .type('{ctrl}')
      .should('have.value', 'hello')
  })
})
```

## 3.4.1 

<img width="923" alt="Screen Shot 2019-10-24 at 9 53 03 AM" src="https://user-images.githubusercontent.com/1271364/67492233-190cf780-f644-11e9-8d63-f1214d99e1d4.png">

## 3.5.0

<img width="923" alt="Screen Shot 2019-10-24 at 9 53 53 AM" src="https://user-images.githubusercontent.com/1271364/67492324-3b9f1080-f644-11e9-9e44-52ec114bc189.png">

This does appear to me to be a bug - since doing this behavior manually does *not* clear the selected value when pressing `{ctrl}` on the keyboard. This may be related to this change: https://github.com/cypress-io/cypress/issues/5424 @Bkucera will have a lot more insight into this however.

**Comment by jennifer-shehane** [comment]
Released in [3.6.0](https://on.cypress.io/changelog#3-6-0)
